flagging_site/data/usgs.py

```diff
@@ -4,8 +4,67 @@
 
 https://waterdata.usgs.gov/nwis/uv?site_no=01104500
 """
+import pandas as pd
+import requests
+import io
 
 # Constants
-# ~~~~~~~~~
+USGS_URL = 'https://waterdata.usgs.gov/nwis/uv?site_no=01104500'
+# Each key is the original column name; the value is the renamed column.
 
-USGS_URL = None
+# ~ ~ ~ ~
+
+def get_usgs_data() -> pd.DataFrame:
+    """This function  runs through the whole process for retrieving data from
+    usgs: first we perform the request, and then we clean the data.
+
+    Returns:
+        Pandas Dataframe containing the usgs data.
+    """
+    res = request_to_usgs()
+    df = parse_usgs_data(res)
+    return df
+
+
+def request_to_usgs(
+) -> requests.models.Response:
+    """
+    Get a request from the USGS.
+
+    Returns:
+        Request Response containing the data from the request.
+    """
+    
+    res = requests.get('https://waterservices.usgs.gov/nwis/iv/?format=json&sites=01104500&period=P7D&parameterCd=00060,00065&siteStatus=all')
+
+    return res
+
+def parse_usgs_data(res) -> pd.DataFrame:
+    """
+    Clean the response from the HOBOlink API.
+
+    Args:
+        res: response object from USGS
+
+    Returns:
+        Pandas DataFrame containing the usgs data.
+    """
+
+    json = res.json()
```

Comment:
I would not name this variable `json`. The issue is that `json` is a base Python package-- one that often gets used alongside `requests` (although not in this case; usually more so for POST requests). So if you name your variables `json`, then it makes it harder down the line to `import json` into the file if we ever need to due to namespace conflicts. I would instead name the variable `data` or `raw_data` or something like that.